crates/core/tedge_agent/src/tedge_to_te_converter/converter.rs

```diff
@@ -0,0 +1,153 @@
+use crate::tedge_to_te_converter::error::TedgetoTeConverterError;
+use clock::Timestamp;
+use log::error;
+use serde::Deserialize;
+use serde::Serialize;
+use serde_json::Value;
+use std::collections::HashMap;
+use std::convert::Infallible;
+use tedge_actors::Converter;
+use tedge_mqtt_ext::MqttMessage;
+use tedge_mqtt_ext::Topic;
+
+#[derive(Debug, Serialize, Deserialize, Eq, PartialEq)]
+pub struct ThinEdgeAlarmData {
```

Comment:
It's not obvious why another struct is introduced, while there is already `tedge_api::alarm::ThinEdgeAlarmData`.

I guess that this is because the former struct doesn't hold a severity.
- What about adding this missing field?
- Is the plan to simplify this in a second step?

Reply:
Yes, I would like to keep this structure temporarily. I will address that issue [here](https://github.com/thin-edge/thin-edge.io/pull/2118) then I will remove this one.

Reply:
> Yes, I would like to keep this structure temporarily. I will address that issue [here](https://github.com/thin-edge/thin-edge.io/pull/2118) then I will remove this one.

I would prefer not.

> I guess that this is because the former struct doesn't hold a severity.

Looking closer, this struct is useless: one just wants to insert a `"severity`" field in a json message. So using serde_json is enough.

I pushed a commit for that https://github.com/thin-edge/thin-edge.io/pull/2125/commits/ba72d24d20881041dd797898b3c6bd952db05344

Reply:
> > Yes, I would like to keep this structure temporarily. I will address that issue [here](https://github.com/thin-edge/thin-edge.io/pull/2118) then I will remove this one.
> 
> I would prefer not.
> 
> > I guess that this is because the former struct doesn't hold a severity.
> 
> Looking closer, this struct is useless: one just wants to insert a `"severity`" field in a json message. So using serde_json is enough.
> 
> I pushed a commit for that [ba72d24](https://github.com/thin-edge/thin-edge.io/commit/ba72d24d20881041dd797898b3c6bd952db05344)

Thank you!.